index.js

```diff
@@ -11,28 +11,75 @@ const nextButton = document.querySelector('[data-js="button-next"]');
 const pagination = document.querySelector('[data-js="pagination"]');
 
 // States
-const maxPage = 1;
-const page = 1;
+let maxPage = 0;
+let page = 1;
 const searchQuery = "";
 
+fetchCharacters("character", page);
+
+// Pagination - final solution
+nextButton.addEventListener("click", () => {
+  page++;
+  prevButton.removeAttribute("disabled");
+  if (page > maxPage) {
+    page = maxPage;
+    nextButton.setAttribute("disabled", "disabled");
+  } else {
+    nextButton.removeAttribute("disabled");
+    fetchCharacters("character", page);
+  }
+  pagination.textContent = `${page} / ${maxPage}`;
+});
+
+prevButton.addEventListener("click", () => {
+  page--;
+  prevButton.removeAttribute("disabled");
+  nextButton.removeAttribute("disabled");
+  pagination.textContent = `${page} / ${maxPage}`;
+  if (page === 1) {
+    prevButton.setAttribute("disabled", "disabled");
+  } else {
+    pagination.textContent = `${page} / ${maxPage}`;
+    fetchCharacters("character", page);
+  }
+});
```

Comment:
Same here, you decrement the page number whenever the button is clicked. But you only want to do that, when the page number is bigger than 1.

```suggestion
prevButton.addEventListener("click", () => {
  if (page > 1) {
    page--;
    fetchCharacters("character", page);
});
```